### PR TITLE
Manga reader updates

### LINKED ISSUE #1820: The image splitting leaves scrollbar on mobile
**Describe the bug**
When I reading double managa, I want to use the image splitting funtion, Whatever option I choose Right to left or Left to right, 
The split images are displayed in full size. This problem shows up on my phone and tablet, But my computer is normal. So I think it's a bug.

**Screenshots**
蜂蜜浏览器_微信图片_20230221081833

蜂蜜浏览器_微信图片_20230221081837

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser: Chrome 110.0
 - Version: Kavita 0.7.1

**Smartphone (please complete the following information):**
 - Device: Mi 10x
 - OS: Android 12
 - Browser: Chrome 110.0
 - Version: Version: Kavita 0.7.1

**Additional context**

### PR DESCRIPTION
**Pull request by therobbiedavis**
# Changed:
- Changed: Changed the ability to toggle emulate comic book on single page layout. This will only add the shadow to wide images.
- Changed: Updated the PR github action to version 10 (develop)

# Fixed:
- Fixed: Fixed some scaling issues with the manga reader. Closes #1820

### CODE REVIEW

**Review by majora2007 — CHANGES_REQUESTED** [review]
As mentioned in chat, this doesn't fix the linked issue and breaks centering vertically fit to screen images.

File: UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts
Comment (majora2007) [inline UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts:34]: You can remove of('') too and use ! as you know it will be setup in constructor

File: UI/Web/src/_manga-reader-common.scss
Comment (majora2007) [inline UI/Web/src/_manga-reader-common.scss:13]: Canvas style should only be in the canvas renderer

File: UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.scss
Comment (majora2007) [inline UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.scss:4]: We already have one in common, you should rework the selector there to accommodate for your single page addition.

File: UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts
Comment (therobbiedavis) [reply UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts:59]: I tried to remove one, but it seems they are both needed otherwise when the toggle is clicked by the user it does not apply.

File: UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts
Comment (therobbiedavis) [reply UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts]: Resolved with https://github.com/Kareadita/Kavita/pull/1892/commits/3adac186e2c7915565094bdd19320380e8a2aabe

File: UI/Web/src/_manga-reader-common.scss
Comment (therobbiedavis) [reply UI/Web/src/_manga-reader-common.scss]: Resolved with https://github.com/Kareadita/Kavita/pull/1892/commits/3adac186e2c7915565094bdd19320380e8a2aabe

**Comment by majora2007** [comment]
This still has bugs, I've handed Robbie files to test on. I cannot merge this unless no side-effect bugs are created.

**Comment by majora2007** [comment]
Since this had issues, I ended up fixing the issue a different way. Please re-raise a separate PR with just your feature request.

**Pull request closed without merging**
